allure-testng/src/main/java/io/qameta/allure/testng/AllureTestNg.java

```diff
@@ -200,11 +204,13 @@ public void onStart(final ITestContext context) {
                 .distinct()
                 .forEach(this::onBeforeClass);
 
-        context.getExcludedMethods().stream()
-                .filter(ITestNGMethod::isTest)
-                .filter(method -> !method.getEnabled())
-                .filter(testFilter::isSelected)
-                .forEach(method -> createFakeResult(context, method));
+        if (config.isHideDisabledTests()) {
```

Comment:
that should be `!config.isHideDisabledTests()`. And why the tests pass?

Reply:
as far as I can see the block actually adds disabled tests to the report. If false condition, disabled tests are not reported

Reply:
Correct - my mistake. 
Fixed, thanks.